### DISCUSSION TITLE
sharing-variables-with-stages-and-jobs

### DISCUSSION THREAD

**Discussion by giscus** [WorkSafeBC-Common-Engineering/Common.Engineering.Discussion#8] [Technical Blog]
# sharing-variables-with-stages-and-jobs

https://wsbctechnicalblog.github.io/sharing-variables-with-stages-and-jobs.html

**Comment by wpschaub** [comment]
The link to "How to share variables amongst Azure Pipeline agents" mentioned at the start of the article has been fixed.
